AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin.swift

```diff
@@ -91,6 +91,13 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
         var token: UnsubscribeToken?
         token = Amplify.Hub.listen(to: .dataStore, isIncluded: filter) { _ in
             self.storageEngine.startSync()
+
+            let payloadOfOutboxStatus = HubPayload(eventName: HubPayload.EventName.DataStore.outboxStatusChanged)
+            Amplify.Hub.dispatch(to: .dataStore, payload: payloadOfOutboxStatus)
+
+            let payload = HubPayload(eventName: HubPayload.EventName.DataStore.ready)
+            Amplify.Hub.dispatch(to: .dataStore, payload: payload)
+
```

Comment:
Once you update the comments explaining when these events are triggered, we may need to discuss why we emit an "outbox status changed" event with no indication as to what the new status is. How is a customer intended to inspect the status of the "outbox" (by which I assume you mean the outgoing mutation queue)?